samples/stock/demo/all-indicators/demo.html

```diff
@@ -0,0 +1,64 @@
+<script src="https://code.highcharts.com/stock/highstock.js"></script>
+<script src="https://code.highcharts.com/stock/modules/data.js"></script>
+<script src="https://code.highcharts.com/stock/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/stock/indicators/indicators-all.js"></script>
+
+<div class="main-wrapper">
+  <div class="selectors-container">
```

Comment:
You can leave it as it is, but do we need this `<div class="selectors-container">`? I don't see any good reasons (maybe there is one) why this
```html
<div class="a">
    <div class="b">
        <p></p>
        <p></p>
        <p></p>
    </div>
</div>
```
cannot be replaced with (with corrected CSS styles and classes maybe)
```html
<div class="a b">
    <p></p>
    <p></p>
    <p></p>
</div>
```

Just a suggestion for the future.

Reply:
`main-wrapper` class is to... well.. wrap the whole selectors and the chart together as it might be a part of some bigger design/site